src/Sylius/Bundle/ApiBundle/Resources/config/serialization/ProductVariant.xml

```diff
@@ -41,5 +41,9 @@
         <attribute name="name">
             <group>shop:product_variant:read</group>
         </attribute>
+        <attribute name="enabled">
+            <group>admin:product_variant:read</group>
+            <group>admin:product_variant:create</group>
```

Comment:
Why did we miss the update part?

Reply:
Because this PR is super old! Gonna fix it.